Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -71,6 +72,20 @@ class NodeController extends ActionController
      */
     protected $propertyMapper;
 
+    /**
+     * Allow invisible nodes to be redirected to
+     *
+     * @return void
+     */
+    protected function initializeShowAction()
```

Comment:
```suggestion
    protected function initializeShowAction(): void
```
do we do those for 4.0 already?

Reply:
I left them out, to be in sync with the existing code.